python/lsst/ctrl/bps/wms_service.py

```diff
@@ -209,6 +209,15 @@ def cancel(self, wms_id, pass_thru=None):
         """
         raise NotImplementedError
 
+    def submission_checks(self):
```

Comment:
Can we change the name of this method to something "verb-like"? Personally, I  think `check()` or `test()` would do just fine. However, if these are too generic for your taste, something like `run_submission_checks` should work too.